patches/chromium/frame_host_manager.patch

```diff
@@ -41,11 +41,24 @@ index 906a1ee4ac58b0744a32153bbaafeac4322a60e4..c90f4aead36cbf3767dc5094728963c2
    // Adds the given SiteInstance to our map, to ensure that we do not create
    // another SiteInstance for the same site.
    void RegisterSiteInstance(SiteInstanceImpl* site_instance);
+diff --git a/content/browser/frame_host/navigation_request.cc b/content/browser/frame_host/navigation_request.cc
+index 496292bd1689607722d4bb27c829d3d7704b9d3c..54bb3a0ebcb2bceb4c1f94f3e38927cf82f039c5 100644
+--- a/content/browser/frame_host/navigation_request.cc
++++ b/content/browser/frame_host/navigation_request.cc
+@@ -1209,6 +1209,8 @@ void NavigationRequest::BeginNavigation() {
+     TRACE_EVENT_ASYNC_STEP_INTO0("navigation", "NavigationRequest", this,
+                                  "ResponseStarted");
+ 
++    state_ = WILL_PROCESS_RESPONSE;
```

Comment:
this is a somewhat terrifying change, perhaps a comment would be good..?

Reply:
Yup will add a comment, this is something that can be upstreamed as well. https://chromium-review.googlesource.com/c/chromium/src/+/1890033 removes it because the condition no longer serves purpose in chromium process model but wouldn't harm anything in having it, but it does make a huge difference for our model, I just have to explain and add it back.

Reply:
Addressed in https://github.com/electron/electron/pull/21000/commits/ab11c507405f96976c0dc9c7d958c99d2ef3a9bf